dpgen/generator/run.py

```diff
@@ -379,6 +383,10 @@ def make_train (iter_index,
         task_path = os.path.join(work_path, train_task_fmt % ii)
         create_path(task_path)
         os.chdir(task_path)
+        
+        if 'srtab_file_path' in jdata.keys():
+            os.system('cp %s ./'%srtab_file_path)
```

Comment:
I suggest using `shutil.copyfile`. `cp` relies on the specific platform (unix).

Reply:
I have modified it.